hoomd/hpmc/update.py

```diff
@@ -444,7 +448,7 @@ def perturb(timestep):
           r = np.sqrt(ext_wall.get_sphere_wall_param(index = 0, param = "rsq"));
           ext_wall.set_sphere_wall(index = 0, radius = 1.5*r,
                                    origin = [0, 0, 0], inside = True);
-        wall_updater = hpmc.update.wall(mc, ext_wall, perturb,
+        wall_updater = hpmc.update.Wall(mc, ext_wall, perturb,
```

Comment:
These examples have the `HPMCIntegrator` object as an argument for the constructor. However, they are not required by `__init__`.